scripts/docs/generatedocs.sh

```diff
@@ -20,7 +20,6 @@ if [ "$(ls -A $DIR/)" ]; then
      echo "Directory $DIR/ must be empty"
      exit 1
 fi
-export PYTHONPATH=$PYTHONPATH:/opt/OpenvStorage/lib/python2.7/site-packages/
```

Comment:
Not sure what this does so better keep it (might have to change once we need this thing)